harbor-helm/templates/NOTES.txt

```diff
@@ -1,2 +1,3 @@
+{{- required "\n\nEULA not accepted.\nPlease review https://documentation.suse.com/beta/eula and to accept, add --set suse.AcceptBetaEULA=y on your helm invocation" .Values.suse.AcceptBetaEULA }}
```

Comment:
I think the error message without the newlines looks much better even comparing to using `fail`

Reply:
ok, removed newlines again